### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by annakocheshkova**
Android properties mapping fix;
Updated specs;
Puppet: should not override properties on child target;
Puppet: moved Other tab and rearranged AppCenter tab.

### CODE REVIEW

**Review by evgeny-pol — DISMISSED** [review]

File: Assets/Puppet/PuppetAppCenter.cs [hunk 1]
```diff
@@ -80,6 +110,19 @@ void OnEnable()
...
+        Push.IsEnabledAsync().ContinueWith(task =>
+        {
+            PushEnabled.isOn = task.Result;
+        });
+        Distribute.IsEnabledAsync().ContinueWith(task =>
+        {
+            DistributeEnabled.isOn = task.Result;
+        });
+
+        Distribute.ReleaseAvailable = (releaseDetails) =>
```
Comment (MatkovIvan) [inline Assets/Puppet/PuppetAppCenter.cs]: `Distribute.ReleaseAvailable = (releaseDetails) => true;`

File: Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Android/AnalyticsInternal.cs [hunk 2]
```diff
@@ -3,7 +3,9 @@
 // Licensed under the MIT license.
 
 #if UNITY_ANDROID && !UNITY_EDITOR
+using Microsoft.AppCenter.Unity.Internal.Utility;
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Android/AnalyticsInternal.cs]: Move `System` usings to top

File: Assets/AppCenter/Plugins/Android/Utility/JavaStringMapHelper.cs [hunk 3]
```diff
@@ -26,10 +28,13 @@ public class JavaStringMapHelper
 
         public static AndroidJavaObject ConvertToJava(IDictionary<string, string> properties)
         {
+            string[] keys = properties.Keys.ToArray();
+            string[] values = properties.Values.ToArray();
+            int count = properties.Count;
             var javaMap = new AndroidJavaObject("java.util.HashMap");
-            foreach (var pair in properties)
+            for (int i = 0; i < count; ++i)
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/Android/Utility/JavaStringMapHelper.cs:35]: Any reason for this?

Comment (annakocheshkova) [reply Assets/AppCenter/Plugins/Android/Utility/JavaStringMapHelper.cs:35]: Yes. The above code doesn't work

Comment (MatkovIvan) [reply Assets/AppCenter/Plugins/Android/Utility/JavaStringMapHelper.cs:35]: Why?

Comment (max-mironov) [reply Assets/AppCenter/Plugins/Android/Utility/JavaStringMapHelper.cs:35]: lets merge it for now as we need to finish current feature in time and current approach will not break anything and we will investigate it later (we can create separate item for this)

**Review by evgeny-pol — APPROVED** [review]

**Pull request merged**